infrastructure/src/test/resources/scenarios/cleanup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# Test init.sh script to run before running scenarios
+echo "Running cleanup.sh test script"
```

Comment:
How do we know cleanup.sh has ran or not in the tests?

Reply:
`set -e` has been added in commit e4af8be to exit the code if a command execution fails.
Modified InfrastructureProvider#init to handle the execution failure and throw an InfrastructureException which will be caught and handled in TestplanExecutor